form.yml

```diff
@@ -50,9 +68,15 @@ cluster:
       $type: text
       $placeholder: /dev/watchdog
       $optional: true
-
+  
+  sbd_checkbox:
```

Comment:
With this configuration the `diskless` option doesn't work. For that, we would need something like:
```
sbd:
  device: false
```
And I don't know if we can achieve this. I don't know if we can have `device` entry working as a list and `boolean` in the same time

Reply:
I could not make this type of configuration work from sumaform.  Maybe a `select` type in form will help here I will check.

Reply:
I had to change the jinja logic in `habootstrap-formula/cluster/create.sls` as mentioned @arbulu89 . However by this, it covers all the usecases